concepts/05 Widgets/Form/01 Getting Started with Form/60 Modify the Form at Runtime.md

```diff
@@ -0,0 +1,212 @@
+You can modify any form, editor or item properties at runtime. In **jQuery**, use the [option(optionName, optionValue)](/Documentation/ApiReference/UI_Widgets/dxForm/Methods/#optionoptionName_optionValue) method to update a property value. In **Angular**, **Vue** or **React**, bind the property's value to a component property.
```

Comment:
Since we have to describe different approaches, let's switch the paragraphs places and divide the description by frameworks as we do in the Create a Form topic.

> You can change any form, editor, or item properties at runtime. The following code dynamically makes the entire Form read-only. %description_for_target_framework%